glide.yaml

```diff
@@ -32,8 +32,8 @@ import:
   version: ^1
 - package: github.com/stretchr/testify
   version: master
-  # TODO: before cherami transport can leave the x/ package, cherami-client-go must >=1.0
 - package: github.com/uber/cherami-client-go
+  version: ^2
```

Comment:
Changed to '>=1, <3'. If any client is using cherami ^1, they should be fine, as the change is only in ^2.